src/tcp/tcp.go

```diff
@@ -208,7 +208,10 @@ func (s *server) clientCommunication(port string, c *comm.Comm) (room string, er
 	}
 	if strings.TrimSpace(string(passwordBytes)) != s.password {
 		err = fmt.Errorf("bad password")
-		enc, _ := crypt.Decrypt([]byte(err.Error()), strongKeyForEncryption)
+		enc, enc_err := crypt.Encrypt([]byte(err.Error()), strongKeyForEncryption)
```

Comment:
Encrypt should be Decrypt?

Reply:
I think `crypt.Decrypt([]byte(fmt.Errorf("bad password").Error()), strongKeyForEncryption)` will always return the `incorrect passphrase` error. Right?
Anything I missed?